librz/search/search.c

```diff
@@ -556,8 +556,20 @@ typedef struct search_ctx {
 	RzSearchOpt *opt; ///< User options
 	RzThreadQueue /* RzSearchHits */ *hits; ///< Hits list
 	RzAtomicBool *loop; ///< If set, the execution will continue until it terminates. If unset, the execution cancels.
+	RzThreadQueue /* RzSearchInterval */ *intervals; ///< Interval list
```

Comment:
```suggestion
	RzThreadQueue /* RzSearchInterval */ *finished_intervals; ///< Interval queue
```

This is the correct interpretation, right?

Reply:
Yes. A complete commit is in 33b30d29a65e036bc9719cd455fc773bbb1a44c8.